### DISCUSSION TITLE
Is it posible to use socket.io-client ?

### DISCUSSION THREAD

**Discussion by avishka-codimite** [react-native-tvos/react-native-tvos#384] [Q&A] [answered]
Hi, Can I know is there a possibility to use the [socket.io-client](https://www.npmjs.com/package/socket.io-client) with **react-native-tvos** ?

**Comment by douglowder** [comment, marked as answer]
I think this should work, since it is Javascript/Typescript only (no native code) and I see that there are already tutorials for making it work with React Native.
